src/main/java/com/elixr/poc/rest/controller/UserDeletionController.java

```diff
@@ -0,0 +1,46 @@
+package com.elixr.poc.rest.controller;
+
+import com.elixr.poc.constants.ApplicationConstants;
+import com.elixr.poc.exception.NoRecordFoundException;
+import com.elixr.poc.rest.response.DeleteResponse;
+import com.elixr.poc.service.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+/*
+RestController for Delete API
+ */
+@RestController
+@RequestMapping("/application")
+public class UserDeletionController {
+
+    private final UserService userService;
+
+    public UserDeletionController(UserService userService) {
+        this.userService = userService;
+    }
+
+    /*
```

Comment:
Recommended comment format is /** */. Modify it in all files

Reply:
Modified.